docs/tutorials/crates/custom_compiled/Makefile

```diff
@@ -0,0 +1,29 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+export CARGO_TARGET_DIR ?= target
+export MODE ?= release
+
+all: clean baked_data/mod.rs
+
+../bin/icu4x-datagen:
+	echo "Target dir: $(CARGO_TARGET_DIR)"
+	ifeq ($(MODE), "release")
+		cargo install icu_datagen --root ..
+	else
+		cargo install icu_datagen --debug --root ..
+	endif
+
+baked_data/mod.rs: ../bin/icu4x-datagen
+	../bin/icu4x-datagen \
+		--format mod \
+		--keys all \
```

Comment:
Suggestion (optional): demonstrate how to build keys specific to the binary

Reply:
Actually iirc key selection does not work for databake because the keys are never used as values and are optimised out.

Reply:
It helps reduce datagen times and compile times. If you don't have enough keys, your code will fail to compile; if you have too many, they should get optimized out, but they still slow down your compilation. But this is an optional suggestion. If you want the standard databake pattern to be building all the data all the time, I'll leave that up to you as the compiled data champion.

Reply:
Oh right but for compiled data you might need to have _something_ there; it could be empty, but we don't have key-specific locale selection.